app/src/main/java/com/example/testapp/ui/movie/MovieActivity.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.testapp.ui.movie
+
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import com.example.testapp.R
+
+
+class MovieActivity : AppCompatActivity() {
```

Comment:
Please make this a fragment. It is not good to have multiple activities for multiple fragments. Host these fragments in single Activity (Please name this Activity to be something meaningful)

Reply:
changed to fragment

Reply:
Still this is an activity instead of fragment